.appveyor.yml

```diff
@@ -20,6 +24,8 @@ clone_folder: *REPO_DIR
 cache:
   - *CACHE_DIR
 
+clone_depth: 50
```

Comment:
Do you mean a build does not restart in this case? Cause Travis fails to get sources as well, but it stats a new build. Does AppVeyor behave differently?